ewcmd.py

```diff
@@ -1128,6 +1182,32 @@ def item_off(id_item, id_server, item_name = "", is_pushed_off = False):
 	return response
 
 
+async def purify(cmd):
+	user_data = EwUser(member=cmd.message.author)
+	
+	if user_data.poi == ewcfg.poi_id_sodafountain:
+		if user_data.life_state == ewcfg.life_state_corpse:
+			response = "You're too ghastly for something like that. Besides, you couldn't even touch the water if you wanted to, it would just phase right through your ghostly form."
+		else:
+			response = "You close your eyes and hold out your hands to the gentle waters of the bicarbonate soda fountain..."
+			
+			user_data.slimelevel = 1
```

Comment:
won't this be a problem with mutations? players will get additional mutations on level-up

Reply:
nevermind, you already fixed it